polybrain.el

```diff
@@ -3,7 +3,7 @@
 
 ;; Author: Dustin Lacewell <dlacewell@gmail.com>
 ;; Version: 0.1.0
-;; Package-Requires: ((emacs "26") (polymode "0") (org-brain "0"))
+;; Package-Requires: ((emacs "26") (polymode "0") (org-brain "0.93"))
```

Comment:
I made some changes in a recent `org-brain` commit (changing version number to 0.93) which allows for the changes in this PR.